specifications/state_sync/README.md

```diff
@@ -0,0 +1,310 @@
+# State Synchronizer specification
+
+## Abstract
+
+This document describes the spec of a Libra node's state synchronizer component.
+
+This document is organized as follows:
+
+1. Overview: high-level outline of the the state synchronizer design and
+functionality.
+2. Sync Protocols: specify technical requirements of various sync protocols 
+recognized in the Libra Network.
+3. Network Protocol: the State Synchronizer LibraNet protocol used to participate
+as a node in the Libra Network
+4. Abstracted Modules: external components that state synchronizer interfaces with
+
+
+## Overview
+
+State Synchronizer is a system component that is responsible for syncing ledger
+state among peers in the Libra Payment Network. Using state sync, a Libra node
+can discover and update to a more recent ledger state if available with the following
+general flow:
```

Comment:
while this is technically correct, it is probably worth stating that it synchronizes the ledger by sharing all transactions and proof that the transactions have been processed by consensus or something. Then state sync re-execute the transactions to actually synchronize the ledger.